### DISCUSSION TITLE
Error: ENOENT: no such file or directory, open '/vercel/path0/.next/server/app/index.rsc'

### DISCUSSION THREAD

**Discussion by Sidney-Dev** [vercel/community#7391] [Help]
### Summary

My builds stopped working even though the site works in my local host. 

Whenever I push to the main branch, the deployment triggers the following error:
    
    Error: ENOENT: no such file or directory, open '/vercel/path0/.next/server/app/index.rsc'

Based on answers from previous questions, I tried rebuilding many times but it still fails. All started after I marged code from a [feature branch](https://github.com/Sidney-Dev/nextjs-dashboard/commit/c77aa81d84ae018880eb50c5435c4ea0c5ce5a17)

[Here](https://github.com/Sidney-Dev/nextjs-dashboard) is my project repo

### Steps to Reproduce

The issue happens whenever I try to run a deployment.

**Comment by jireve999** [comment]
I met the same issue and it works fine after removed experimental feature.

[a comment by an automated account is not shown]
